articles/user-guide/consolidating-accounts.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Consolidating/Migrating SparkPost Accounts"
-description: "Subaccounts As you start utilizing subaccounts it will be essential to become more familiar with how management will work with this feature Please read through this article before determining how you would like to set up your account u Subaccounts in Spark Post and Spark Post Elite u Determine what..."
+description: "Subaccounts As you start utilizing subaccounts it will be essential to become more familiar with how management will work with this feature Please read through this article before determining how you would like to set up your account u Subaccounts in SparkPost and SparkPost Enterprise u Determine what..."
```

Comment:
- Pls remove odd `Subaccounts` from the start of description.
 - Plz use full stops between sentences.
 - Remove the last sentence entirely? Looks like it has suffered lots of edits and no longer makes sense.